src/firefly/js/ui/tap/SpatialSearch.jsx

```diff
@@ -78,7 +79,7 @@ export function SpatialSearch({cols, serviceUrl, columnsModel, tableName, initAr
     const {canUpload=false}= capabilities ?? {};
 
     const {setConstraintFragment}= useContext(ConstraintContext);
-    const {setVal,getVal,makeFldObj,groupKey}= useContext(FieldGroupCtx);
+    const {setVal,getVal,setFld,makeFldObj,groupKey}= useContext(FieldGroupCtx);
```

Comment:
Looks like you decided not to use `setFld` you should remove it. I don't think `groupKey` is used anymore here either.

Reply:
Yes, setFld didn't allow me to send a value (to set the initial lon/lat values) so I ended up using setVal. Will remove setFld and groupKey and merge.

Reply:
What you are doing is find. with `setFld` you just include include the object `{value:v, other stuff}`